scripts/linters/pylint_extensions.py

```diff
@@ -165,6 +174,24 @@ def visit_call(self, node):
                             func_name))
                     num_positional_args_unused -= 1
 
+        for arg in keyword_args:
+            # If there is *args and **kwargs in the functin definition skip the
+            # check because we can use keywords arguments in function call even
+            # if **kwargs is present in the function definition.
+            if not called.args.kwarg and callable_name != 'constructor':
+                if not arg in keyword_args_in_funcdef:
+                    # This try/except block tries to get the function
+                    # name. Since each node may differ, multiple
+                    # blocks have been used.
```

Comment:
Didn't get this `Since each node may differ, multiple blocks have been used.`

Reply:
Sorry, this is just saying we used two blocks here, i.e. try and except. I'll remove these lines as this is creating confusion